### ISSUE TITLE
Cannot use inside Animated.FlatList 

### ISSUE THREAD

**Issue by morajabi** [dominicstop/react-native-ios-context-menu#79]
It opens correctly but on close it crashes the app.

https://github.com/dominicstop/react-native-ios-context-menu/assets/12202757/73e821d5-682e-4ac6-bc55-2b1fc967e9d9

<img width="311" alt="CleanShot 2023-11-04 at 13 18 31@2x" src="https://github.com/dominicstop/react-native-ios-context-menu/assets/12202757/5056c2e5-abf0-47b7-be87-5ea761f80d69">

**Comment by nandorojo** [comment]
Possible duplicate or #70

**Comment by nandorojo** [comment]
Have you tried FlashList

**Comment by nandorojo** [comment]
And are you on RN 0.72.5+

**Comment by morajabi** [comment]
Unfortunately FlashList falls short for inverted usecase, messages frequently overlap over each other and it doesn't support sticky items properly while inverted. We recently had to switch to FlatList.

**Comment by nandorojo** [comment]
And making FlashList not be animated fixes it?

**Comment by morajabi** [comment]
@nandorojo Yes. But also I noticed a new thing. Clicking on of our menu items caused the crash, but in production it was fine.

**Comment by nandorojo** [comment]
In my case, quitting the dev app and restarting fixed this

**Comment by morajabi** [comment]
I'll try again with Animated FlatList with this suggestion
